kolibri/plugins/learn/assets/src/views/content-card-grid/index.vue

```diff
@@ -80,6 +76,7 @@
       genLink: {
         type: Function,
         validator(value) {
+          // callback: function(contentId, contentKind)
```

Comment:
It was actually supposed to be a short line of documentation heh. I'll clear it out though